Mini/MiniCooperSE.json

```
@@ -0,0 +1,71 @@
+{
+  "init_commands": {
+    "command": [
+	"ATZ", 
+	"ATD", 
+	"ATE0", 
+	"ATS0", 
+	"ATAL",
+	"ATPBE101",
+	"ATSPB",
+	"ATBI",
+	"ATSH6F1",
+	"ATAT0",
+	"ATSTFF",
+	"ATCRA607",
+        "ATCEA07",
+        "ATFCSH6F1",
+        "ATFCSD07300800",
+        "ATFCSM1"
+	]
+  },
+  "data_commands": {
+    "command": [
+      "22DDBC",
+      "22DD68",
+      "22DD69",
+      "226335"
+    ]
+  },
+  "obd_protocol": "B",
+  "soc": {
+    "equation": "(G*256+H)*0.1",
+    "minValue": "0",
+    "maxValue": "100",
+    "type": "Number",
+    "command": "22DDBC",
+    "ecu": "607"
+  },
+  "voltage": {
+    "equation": "(F*256+G)*0.01",
+    "minValue": "300",
+    "maxValue": "450",
+    "type": "Number",
+    "command": "22DD68",
+    "ecu": "607"
```

I think the ECU normall iy the adress you use on ATSH commands. ATSH is the send header and ACRA the response header. So ATSH describes the eCU where the commands should be sent to:
`ATSH6F1`
```suggestion
    "ecu": "6F1"
```

I might be wrong though but at least this is what we did for most other cars. Not so super important right now, as we do not use it, but in future we want to generically create init and data commands, so then those values should be correct.

---

Oh ok, I'm pretty sure the ecu number is 607, I don't really know where the 6F1 came from. I just snooped the ATcommands when using an app and then incorporated that info into my own arduino and iOs project. But if it can be improved that would be better. You have way more experience in that. :)

---

I'm not a pro expert on that field myself, will check again. In the end it doesn't matter so much as long as we use the correct value ;).